File: PB_Online_Exams_July/Best_Player.py
import sys

max_score = -sys.maxsize
name = ""
max_name = ""

current_name = input()
while current_name != "END":
    goals = int(input())
    name = current_name

    if goals > max_score:
        max_score = goals
        max_name = name

    if goals >= 10:
        break

    name = input()

print(f'{max_name} is the best player!')

if goals >= 3:
    print(f'He has scored {goals} goals and made a hat-trick !!!')
else:
    print(f'He has scored {goals} goals.')

File: C++/Exam/BestPlayer/BestPlayer.cpp
#include <iostream>
#include <string>
#include <climits>

using namespace std;

int main()
{
	int maxScore = INT_MIN;
	string name = "";
	string maxName = "";
	string currentName;
	int goals;

	getline(cin >> ws, currentName);

	while (currentName != "END")
	{
		cin >> goals;
		name = currentName;

		if (goals > maxScore)
		{
			maxScore = goals;
			maxName = name;
		}

		if (goals >= 10)
		{
			break;
		}

		getline(cin >> ws, currentName);
	}
	cout << maxName << " is the best player!" << endl;

	if (goals >= 3)
	{
		cout << "He has scored " << maxScore << " goals and made a hat-trick !!!" << endl;
	}
	else
	{
		cout << "He has scored " << goals << " goals." << endl;
	}
}